plugins/woocommerce/tests/e2e-pw/tests/merchant/order-edit.spec.js

```diff
@@ -42,16 +42,20 @@ test.describe( 'Edit order', () => {
 	} );
 
 	test( 'can view single order', async ( { page } ) => {
-		// go to orders page
-		await page.goto( 'wp-admin/edit.php?post_type=shop_order' );
+		if ( process.env.DISABLE_HPOS === 1 ) {
+			await page.goto( 'wp-admin/edit.php?post_type=shop_order' );
+		} else {
+			await page.goto( '/wp-admin/admin.php?page=wc-orders&action=new' );
```

Comment:
This url is wrong, it should not contain `action=new`. If the above condition is fixed this will make the test fail, because it expects the Orders page while this url is for new order page.

```suggestion
			await page.goto( 'wp-admin/admin.php?page=wc-orders' );
```